src/model/NPMResolver.ts

```diff
@@ -67,7 +67,7 @@ export class NPMResolver {
      */
     private static installPluginDependencies(pluginName: string, assetsPath: string): boolean {
         console.log(`⟲ [${pluginName}] (NPM) installing dependencies...`);
-        const res = spawn.sync('npm', ['install', '--prefix', assetsPath]);
+        const res = spawn.sync('npm', ['install', '--prefix', assetsPath, '--no-save']);
```

Comment:
This solved my issue. Link is not created anymore. A `./gradlew generatePluginDescriptor build -x test --parallel --stacktrace` was successful. ✔️

Reply:
The presence of the symlink does not fail the build on Linux or Mac, it seems. But based on what @Kaltenbach explained, it prolongs build times because it causes a re-hash of the whole repository. So once the new version becomes available, you (@slaven3kopic) should announce at least in `#devtalk` that the symlink should be removed if present.